src/components/Form.jsx

```diff
@@ -25,18 +25,20 @@ function Form() {
   return (
     <form onSubmit={handleSubmit}>
       <input
+        className="bookTitle"
         type="text"
         placeholder="Book Title"
         value={title}
         onChange={(e) => setTitle(e.target.value)}
       />
       <input
+        className="bookAuthor"
         type="text"
         placeholder="Author"
         value={author}
         onChange={(e) => setAuthor(e.target.value)}
       />
-      <button type="submit">Add Book</button>
+      <button className="submitButton" type="submit">Add Book</button>
     </form>
```

Comment:
- [ ] 👍 Good work so far! However, there's a minor layout issue. As per the design, your input fields should take up the full width until the button, and the button should be aligned horizontally with the card on the right side. Currently, they are not aligned as intended. Please update your code to ensure that the layout matches the design, providing a more polished and visually consistent appearance. Keep it up! 🛠️👏🎨

| Your | Design |
| --- | --- |
|  ![image](https://github.com/eamador1/bookstore-redux/assets/55840999/e12096ba-da3a-40f0-8db9-0669ab3fab69)| ![image](https://github.com/eamador1/bookstore-redux/assets/55840999/e8ac791c-3425-4dc6-9d30-96f4dac85cf6)|